zigbee: fix issues with aborted FOTA downloads (#14684)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_zigbee` protocol.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0

NCSIDB-1213: Subsequent Zigbee FOTA updates fail

>   Once a Zigbee FOTA update is interrupted for any reason, subsequent ones will fail until a device reboot since :ref:`lib_dfu_target` resources are not freed.

Reviewer: ```suggestion
  Once a Zigbee FOTA update is interrupted for any reason, the subsequent updates will fail until a device reboot.
  This is because :ref:`lib_dfu_target` resources are not freed.
```
Will this be fixed? If not, you can apply the `Won't Fix` tag. See the top of the known_issues.rst file for info.

Author: This PR fixes this issue, but previous releases are affected.
